Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -280,7 +317,9 @@ file that was distributed with this source code.
             {% endif %}
         {% endif %}
 
-        {{ form_label(form, label|default(null)) }}
+        {% if not hide_label %}
```

Comment:
When possible, avoid negative notions when naming boolean. This is kind of a double negation, and `{% if show_label %}` would be more natural, wouldn't it?